File: Tareas Java/Paralelogramo1/Paralelogramo1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Paralelogramo
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        int Base, altura;
        FormDibujo formularioDibujo = new FormDibujo();

        private void btnDibujar_Click(object sender, EventArgs e)
        {
            Base = Convert.ToInt32(txtBase.Text);
            altura = Convert.ToInt32(txtAltura.Text);

            if (Base < 3 || altura < 3) {
                //scribir "No se puede dibujar el trapecio";
            }
            else {
                this.Hide();
                formularioDibujo.Base = Base;
                formularioDibujo.altura = altura;
                formularioDibujo.Show();
            }
        }
    }
}

File: Tareas Java/Paralelogramo/Paralelogramo/Form1.vb
Public Class Form1

    Dim base, altura As Integer
    Dim formularioDibujo As New FormDibujo()

    Private Sub btnDibujar_Click(sender As Object, e As EventArgs) Handles btnDibujar.Click

        base = Convert.ToInt32(txtBase.Text)
        altura = Convert.ToInt32(txtAltura.Text)

        If base < 3 Or altura < 3 Then
            'scribir "No se puede dibujar el trapecio";
        Else
            Me.Hide()
            formularioDibujo.base = base
            formularioDibujo.altura = altura
            formularioDibujo.Show()
            'DibujarParalelogramo(base, altura)
        End If
    End Sub
End Class
